wire-library/wire-kotlin-generator/src/main/java/com/squareup/wire/kotlin/KotlinGenerator.kt

```diff
@@ -1328,7 +1329,8 @@ class KotlinGenerator private constructor(
         .addSuperclassConstructorParameter("\n%S", type.type.typeUrl!!)
         .addSuperclassConstructorParameter("\n%M",
             MemberName(Syntax::class.asClassName(), type.syntax.name))
-        .addSuperclassConstructorParameter("\nnull\n⇤")
+        .addSuperclassConstructorParameter("\nnull⇤")
```

Comment:
nit: can you move the `⇤` to the next line, after the `\n` character? That’ll fix the issue where it’s unindented one line too early.

Reply:
Fixed.